packages/jobs/lib/runner/kubernetes.ts

```diff
@@ -1,35 +1,58 @@
 import * as k8s from '@kubernetes/client-node';
 
-import { getPersistAPIUrl, getProvidersUrl } from '@nangohq/shared';
 import { Err, Ok } from '@nangohq/utils';
 
 import { envs } from '../env.js';
 
 import type { Node, NodeProvider } from '@nangohq/fleet';
 
+// Load Kubernetes config (works with local kubeconfig or in-cluster config)
+const kc = new k8s.KubeConfig();
+kc.loadFromDefault();
+const defaultNamespace = envs.RUNNER_NAMESPACE;
+const namespacePerRunner = envs.NAMESPACE_PER_RUNNER || false;
+const appsApi = kc.makeApiClient(k8s.AppsV1Api);
+const coreApi = kc.makeApiClient(k8s.CoreV1Api);
```

Comment:
Can this instantiation be made lazily? otherwise, it will always trigger even locally for nothing